README.md

```diff
@@ -1,5 +1,18 @@
 # OData .NET Libraries
-<img src="https://identitydivision.visualstudio.com/_apis/public/build/definitions/2cfe7ec3-b94f-4ab9-85ab-2ebff928f3fd/106/badge"/>
+<table>

```

Comment:
You can use the special mark syntax for table.

See #Table at https://guides.github.com/features/mastering-markdown/